### PR TITLE
Release notes for the 0.3.2 alpha server release

### LINKED ISSUE #1301: Add release notes for the 0.3.2 alpha release
This release includes changes for metadata api support and searching features by name and gene_symbol

### PR DESCRIPTION
**Pull request by kozbo**
Adding Metadata and the ability to search features on name or gene_symbol
Fixes #1301

### CODE REVIEW

File: docs/status.rst [hunk 1]
```diff
@@ -20,6 +20,20 @@ Release Notes
 +++++++++++++
 
 *****
+0.3.2
+*****
+
+Alpha pre-release supporting major feature update.
+
+- Metadata functionality added to support biodata and individual metadata
```
Comment (david4096) [inline docs/status.rst]: *biosample

**Comment by kozbo** [comment]
@david4096 - updated

**Comment by david4096** [comment]
+1

**Comment by jeromekelleher** [comment]
+1

**Pull request merged**